apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -389,7 +389,7 @@
       ...mapState('tabs', { 'activeTab': 'active' }),
       userKeymap: {
         get() {
-          const value = this.settings?.keymap.value;
+          const value = this.settings?.keymap?.value;
```

Comment:
I have a PR that will fix the `keymap` from returning null or undefined so it should be fine without the `?`. But It's fine with it too regardless.